src/styles/_variables.scss

```diff
@@ -1,26 +1,29 @@
-$primary-color: #1DA1F2;
-$secondary-color: #14171A;
-$background-color: #FFFFFF;
-$text-color: #14171A;
-$gray-light: #E1E8ED;
+$primary-color: #1da1f2;
+$secondary-color: #14171a;
+$background-color: rgba(182, 178, 178, 0.1);
+$gray-light: #e1e8ed;
 $gray-dark: #657786;
-$error-color: #E0245E;
-$success-color: #17BF63;
+$error-color: #e0245e;
+$success-color: #17bf63;
+$black: #000000;
+$border-color: rgba(182, 178, 178, 0.1);
+$text-color: #ffffff;
+$input-background-color: rgba(182, 178, 178, 0.01);
 
 // Font sizes
-$font-xs: 0.75rem;    // 12px
-$font-sm: 0.875rem;   // 14px
-$font-base: 1rem;     // 16px
-$font-lg: 1.125rem;   // 18px
-$font-xl: 1.25rem;    // 20px
-$font-2xl: 1.5rem;    // 24px
+$font-xs: 0.75rem; // 12px
+$font-sm: 0.875rem; // 14px
+$font-base: 1rem; // 16px
+$font-lg: 1.125rem; // 18px
+$font-xl: 1.25rem; // 20px
+$font-2xl: 1.5rem; // 24px
 
 // Spacing
-$spacing-xs: 0.25rem;   // 4px
-$spacing-sm: 0.5rem;    // 8px
-$spacing-md: 1rem;      // 16px
-$spacing-lg: 1.5rem;    // 24px
-$spacing-xl: 2rem;      // 32px
+$spacing-xs: 0.25rem; // 4px
+$spacing-sm: 0.5rem; // 8px
+$spacing-md: 1rem; // 16px
+$spacing-lg: 1.5rem; // 24px
+$spacing-xl: 2rem; // 32px
```

Comment:
anywhere you are using gap or margin or padding these are the values you should use not the hard coded ones and use the same logic as i mentioned above for font size to choose which size to use from these

Reply:
fixed all (gap, margin,  and padding) from hard coded to variables